python/plugins/processing/core/GeoAlgorithm.py

```diff
@@ -213,8 +213,16 @@ def execute(self, progress=None, model=None):
             lines = [self.tr('Uncaught error while executing algorithm')]
             lines.append(traceback.format_exc())
             ProcessingLog.addToLog(ProcessingLog.LOG_ERROR, lines)
+            try:
+                message = unicode(e)
+            except Exception:
```

Comment:
Should this check for a `UnicodeDecodeError`? I don't expect much trouble here, but in the past, generic exception catching has unintentionally masked other issues.